src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -1084,15 +1087,15 @@ export default {
 	}
 }
 
+.left-sidebar__settings-button-container {
```

Comment:
You might now revert this commit: [9b21e9dc9d14bd8549086861ee663349ffffde8d](https://github.com/nextcloud/spreed/commit/9b21e9dc9d14bd8549086861ee663349ffffde8d)